test/index.html

```diff
@@ -1,55 +1,153 @@
-    </ol>
-  </section>
-  <section>
-    <h2>Navigation: Only a single navigation element should exist</h2>
-    <nav>This is the first navigation</nav>
-    <nav>This is the second navigation</nav>
-  </section>
-  <section>
-    <h2>HTML: html tag should have a lang attribute</h2>
-  </section>
-  <section>
-    <h2>Description list: Direct children of a &lt;dl&gt; can only be &lt;dt&gt; and &lt;dd&gt;</h2>
-    <dl>
-      <dt>Hello, I'm dt element inside dl element</dt>
-      <dd>Hello, I'm dd element inside dl element</dd>
-      <p>Hello, I'm p(paragraph) element inside dl element. </p>
-    </dl>
-  </section>
-  <section>
-    <h2>External Links: Provide advanced warning to a user when a link opens in a new window or tab</h2>
-    <a href="https://github.com/jackdomleo7/Checka11y.css" target="_blank">Checka11y Repo</a>
-  </section>
-</body>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Checka11y.css test</title>
+    <link rel="stylesheet" href="./index.css" />
+    <link rel="stylesheet" href="../checka11y.css" />
+  </head>
+  <body>
+    <h1>Checka11y.css - test.html</h1>
+    <p>
+      The intention of this file is to test out the new a11y checks you are adding, so this web page should try to not 
+      comply with a11y as much as possible.</p>
+    <hr />
+    <section>
+      <h2>Images: All images require an alt attribute, whether it has a value or is null</h2>
+      <img src="./assets/a11y.png" />
+    </section>
+    <section>
+      <h2>iframe: iframe has title attribute</h2>
+      <iframe src="https://jackdomleo.dev" height="500" width="500"></iframe>
+    </section>
+    <section>
+      <h2>Lists: All children of &lt;ol&gt; must only be &lt;li&gt;</h2>
+      <ol>
+        <div>Hello, I'm a div inside a &lt;ol&gt;</div>
+      </ol>
+    </section>
+    <section>
+      <h2>Lists: All children of &lt;ul&gt; must only be &lt;li&gt;</h2>
+      <ol>
+        <div>Hello, I'm a div inside a &lt;ul&gt;</div>
+      </ol>
+    </section>
+    <section>
+      <h2>Navigation: Only a single navigation element should exist</h2>
+      <nav>This is the first navigation</nav>
+      <nav>This is the second navigation</nav>
+    </section>
+    <section>
+      <h2>HTML: html tag should have a lang attribute</h2>
+    </section>
+    <section>
+      <h2>Description list: Direct children of a &lt;dl&gt; can only be &lt;dt&gt; and &lt;dd&gt;</h2>
+      <dl>
+        <dt>Hello, I'm dt element inside dl element</dt>
+        <dd>Hello, I'm dd element inside dl element</dd>
+        <p>Hello, I'm p(paragraph) element inside dl element. </p>
+      </dl>
+    </section>
+    <section>
+      <h2>External Links: Provide advanced warning to a user when a link opens in a new window or tab</h2>
+      <a href="https://github.com/jackdomleo7/Checka11y.css" target="_blank">Checka11y Repo</a>
+    </section>
+    <section>
```

Comment:
I consider this entire PR to be one feature, so this is my bad for not clearly stating it (I'll improve the docs later) but could you move all this into one `<section>`.

`<h2>Checks for invalid HTML elements nested inside an &lt;a&gt;><h2>`

Then list your current h2s as `<h3>`.

Reply:
Should I just wrap all these section with a section 
```
<section>
    <h2></h2>
    <section>
        <h3></h3>
        <a>
            <audio>
        </a>
    </section>
    <section>
        <h3></h3>
        <a>
            <video>
        </a>
    </section>
</section>
```

or should I remove the sections from everywhere and just keep one section tag.

```
<section>
    <h2></h2>
    <h3></h3>
    <a>
        <audio>
    </a>
    <h3></h3>
    <a>
        <video>
    </a>
</section>
```

Reply:
The latter please, remove all sections but leave one 😊